src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -130,9 +144,7 @@ public TargetingEvaluationOptions AssignerOptions
         /// <returns>True if the feature is enabled, otherwise false.</returns>
         public async Task<bool> IsEnabledAsync(string feature)
         {
-            EvaluationEvent evaluationEvent = await EvaluateFeature<object>(feature, context: null, useContext: false, CancellationToken.None);
-
-            return evaluationEvent.Enabled;
+            return (await EvaluateFeature<object>(feature, context: null, useContext: false, CancellationToken.None).ConfigureAwait(false)).Enabled;
```

Comment:
Yes it was not there. Although it felt safe to me to add it- because the function it calls (EvaluateFeature) uses `.ConfigureAwait(false)` when reading configs and when evaluating true or false. So I don't think anyone could have utilized the default (aka `.ConfigureAwait(true)`) behavior.